src/main/java/seedu/address/ui/LoginScreen.java

```diff
@@ -60,15 +61,24 @@ private void handleUserInputPassword() {
     }
 
     private void handlePassword() {
+        // wrong format
+        if (!PasswordCommandParser.isValidPassword(userInputPassword.getText())) {
+            responseDisplay.setText("Wrong password, try again!");
+            userInputPassword.clear();
+            return;
+        }
```

Comment:
Not sure if this is block is necessary anymore.

Reply:
This blocks the case when you append `/` at the back and still manage to log in.

Reply:
Ah ok. Can add an explanatory comment here.